src/main/java/de/dafuqs/spectrum/registries/SpectrumBlocks.java

```diff
@@ -2259,13 +2286,28 @@ private static void registerStoneBlocks(FabricItemSettings settings) {
 		registerBlockWithItem("longing_chimera", LONGING_CHIMERA, settings, DyeColor.BROWN);
 
 		registerBlockWithItem("pyrite", PYRITE, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_slab", PYRITE_SLAB, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_stairs", PYRITE_STAIRS, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_wall", PYRITE_WALL, settings, DyeColor.BROWN);
 		registerBlockWithItem("pyrite_pile", PYRITE_PILE, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_pile_slab", PYRITE_PILE_SLAB, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_pile_stairs", PYRITE_PILE_STAIRS, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_pile_wall", PYRITE_PILE_WALL, settings, DyeColor.BROWN);
 		registerBlockWithItem("pyrite_tiles", PYRITE_TILES, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_tiles_slab", PYRITE_TILES_SLAB, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_tiles_stairs", PYRITE_TILES_STAIRS, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_tiles_wall", PYRITE_TILES_WALL, settings, DyeColor.BROWN);
 		registerBlockWithItem("pyrite_plating", PYRITE_PLATING, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_plating_slab", PYRITE_PLATING_SLAB, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_plating_stairs", PYRITE_PLATING_STAIRS, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_plating_wall", PYRITE_PLATING_WALL, settings, DyeColor.BROWN);
 		registerBlockWithItem("pyrite_tubing", PYRITE_TUBING, settings, DyeColor.BROWN);
 		registerBlockWithItem("pyrite_relief", PYRITE_RELIEF, settings, DyeColor.BROWN);
 		registerBlockWithItem("pyrite_stack", PYRITE_STACK, settings, DyeColor.BROWN);
-		registerBlockWithItem("pyrite_panneling", PYRITE_PANNELING, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_paneling", PYRITE_PANELING, settings, DyeColor.BROWN);
+		registerBlockWithItem("pyrite_paneling_slab", PYRITE_PANELING_SLAB, settings, DyeColor.BROWN);
```

Comment:
Do we really need variants for everything?
Especially stairs and walls take up a LOT of block states

Reply:
i mean, i was only thinking of doing planed calcite and basalt and then azzyy challenged me to do pyrite as well in insider trading. i can remove this if need be